setup.py

```diff
@@ -27,6 +27,7 @@
 
 
 def extensions(debug=False, use_cython=True):
+    # TODO: remove fast-math?
```

Comment:
yes, but that's better done in a separate PR — I totally agree with adding the comment here